rdkit/Dbase/UnitTestDbConnect.py

```diff
@@ -207,7 +207,7 @@ def test_InsertData(self):
 
     self.assertEqual(len(conn.GetColumnNames(table=newTblName)), 3)
     conn.AddColumn(newTblName, 'val3', 'int')
-    #     conn.Commit()
+    conn.Commit()
```

Comment:
I wonder why it was commented out. If there was higher priority for `DbConnect`, I would suggest adding a context manager for transactions. Something like this.
```
@contextmanager
def transaction(conn):
  try:
    yield
  exception:
    conn.cn.rollback() ?
  else:
    conn.Commit()
```
But that is a different PR.